.circleci/config.yml

```diff
@@ -50,6 +50,9 @@ jobs:
                   name: Lint Twig
                   command: php bin/console lint:twig templates/
                   when: always
+            - run:
+                    name: Run PHPStan
+                    command: php -d memory_limit=-1 vendor/bin/phpstan analyse -c ./phpstan.neon.dist src
```

Comment:
src is uselss here since it's already in the config file. 
BTW, the config file option is useless too, since it's the default value
And Renamed the config file to `phpstan.neon`